libs/ui/src/lib/theme.tsx

```diff
@@ -1,10 +1,38 @@
 import {createTheme} from '@mui/material'
 
 export const theme = createTheme({
+  typography: {
+    fontFamily: ['Hanken Grotesk', 'Roboto', 'sans-serif'].join(','),
+    h4: {
```

Comment:
Were the font-sizes of h4/h6 changed because of the navigation? Probably not ideal as those are mostly for content.  This now looks weird inside article content.

<img width="836" alt="image" src="https://github.com/wepublish/wepublish/assets/1131901/a1c60b41-79cd-44e2-88f7-43a77ba68bb6">